_opcodes/op006.html

```diff
@@ -15,7 +15,13 @@
 <br/>0   Cumulative Modifier -> Charisma = Charisma + 'Statistic Modifier' value
 <br/>1   Flat Value Modifier -> Charisma = 'Statistic Modifier' value
 <br/>2   Percentage Modifier -> Charisma = (Charisma * 'Statistic Modifier' value) / 100
-<br/>'Statistic Modifier' can be positive or negative.
+
+{% capture note %}
+'Statistic Modifier' can be positive or negative.
+{% endcapture %}
+
+{% include info.html %}
+
```

Comment:
These seem excessive. While it is true that the statement does not hold for all opcodes, I don't think it is so important to be displayed so prominently.